Solutions/Corvus.Monitoring.ApplicationInsights.Specs/Corvus.Monitoring.ApplicationInsights.Specs.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$(EndjinProjectPropsPath)" Condition="$(EndjinProjectPropsPath) != ''" />
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6</TargetFramework>
```

Comment:
According to https://learn.microsoft.com/en-us/dotnet/standard/frameworks the canonical TFM for .NET 6 is `net6.0`.